### PR TITLE
composer: simplify autoloading paths to make it more readable

### PR DESCRIPTION
**Pull request by TomasVotruba**
| Q             | A
| ------------- | ---
|Description, reason for the PR| Cleaner reading of `composer.json`
|New feature| No
|BC breaks| No
|Standards and tests pass| Yes
|License| MIT
|Have you read and signed our [License Agreement for contributions](https://www.shopsys-framework.com/license-agreement)?| Yes

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -9,36 +9,16 @@
...
-      ],
-      "Shopsys\\ProductFeed\\HeurekaDeliveryBundle\\": [
-        "packages/product-feed-heureka-delivery/src/"
-      ],
-      "Shopsys\\FrameworkBundle\\": [
-        "packages/framework/src/"
-      ]
+      "Shopsys\\FormTypesBundle\\": "packages/form-types-bundle/src/",
+      "Shopsys\\HttpSmokeTesting\\": "packages/http-smoke-testing/src/",
+      "Shopsys\\MigrationBundle\\": "packages/migrations/src/Shopsys/MigrationBundle/",
```
Comment (TomasVotruba) [inline composer.json]: Btw, now I see this directory structure is different to all others. Why is that?

I'd expect this:

```json
   "Shopsys\\MigrationBundle\\": "packages/migrations/src/",
```

Comment (TomasVotruba) [reply composer.json]: I see. What is the reason to keep it this way? 
I think [monorepo-tools](https://github.com/shopsys/monorepo-tools) handles this easily

**Comment by TomasVotruba** [comment]
@PetrHeinz I forgot those. Done :+1:

**Comment by PetrHeinz** [comment]
@TomasVotruba thanks, merging it into master :)

**Comment by TomasVotruba** [comment]
:+1:  You're welcome!

**Pull request merged**